app/app.go

```diff
@@ -440,6 +547,8 @@ func NewDesmosApp(
 		profilestypes.ModuleName,
 
 		crisistypes.ModuleName,
+
+		wasm.ModuleName,
```

Comment:
As above, I would move this after the `feegrant.ModuleName`. Also because, as written inside the node: 
```
// NOTE: The crisis module should occur after all other modules to run the invariants at genesis correctly
```